05-Jason-Luo/wk02/4-thu/mad_libs-jquery/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+
+<body>
+
+    <h1>Mad Libs</h1>
+
+    <ul>
+        <li>Noun: <input type="text" id="noun">
```

Comment:
```suggestion
        <li>Noun: <input id="noun">
```
Text is the default type of input field so there's no need to specify `type="text"`.